README.md

```diff
@@ -28,6 +28,8 @@ To further configure your plugin, use the following custom parameters in your `s
 | `site`                 | Set which Datadog site to send data, this is only used when `flushMetricsToLogs` is `false` or `addExtension` is `true`. Possible values are `datadoghq.com`, `datadoghq.eu`, `us3.datadoghq.com` and `ddog-gov.com`. The default is `datadoghq.com`.                                                                                                                                                               |
 | `apiKey`               | Datadog API Key, only needed when `flushMetricsToLogs` is `false` or `addExtension` is `true`. For more information about getting a Datadog API key, see the [API key documentation][3].                                                                                                                                                                                                                            |
 | `apiKMSKey`            | Datadog API Key encrypted using KMS. Use this parameter in place of `apiKey` when `flushMetricsToLogs` is `false` or `addExtension` is `true`, and you are using KMS encryption.                                                                                                                                                                                                                                    |
+| `monitorsApiKey`       | Datadog API Key. Only needed when using plugin to create monitors for your function and when `monitors` is defined. Separate from  `apiKey` with your function,  `monitorsApiKey` is only used to create monitors through the Datadog Monitors API.                                                                                                                                                                                                            |
```

Comment:
I would clarify this a bit more:
- You can use the same API key for `apiKey` and `monitorsApiKey` if you want
- You mentioned "your function" but the user could have multiple functions